nodes_within_range.py

```diff
@@ -0,0 +1,11 @@
+#Can you augment a BST to return the number
+#of elements with node values in a given range?
+def nodes_within_range(tree, value1, value2, n):
+                if tree != None:
```

Comment:
4 spaces, not 20.

    if tree is not None

For signleton objects like None, False, True always check "is". Is checks reference to the same place in memory, while == checks for value. True == 1 is true, True is 1 is false. Only None is None.